src/System.Reflection.DispatchProxy/src/CompatibilitySuppressions.xml

```diff
@@ -0,0 +1,54 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<!-- https://learn.microsoft.com/en-us/dotnet/fundamentals/package-validation/diagnostic-ids -->
+<Suppressions xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema">
+  <Suppression>
+    <DiagnosticId>CP0003</DiagnosticId>
+    <Target>System.Reflection.DispatchProxy, Version=4.0.6.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51</Target>
+    <Left>lib/net461/System.Reflection.DispatchProxy.dll</Left>
+    <Right>lib/netstandard2.0/System.Reflection.DispatchProxy.dll</Right>
+    <IsBaselineSuppression>true</IsBaselineSuppression>
+  </Suppression>
+  <Suppression>
+    <DiagnosticId>CP0003</DiagnosticId>
+    <Target>System.Reflection.DispatchProxy, Version=4.0.6.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51</Target>
+    <Left>lib/netcoreapp2.0/System.Reflection.DispatchProxy.dll</Left>
+    <Right>lib/netstandard2.0/System.Reflection.DispatchProxy.dll</Right>
+    <IsBaselineSuppression>true</IsBaselineSuppression>
+  </Suppression>
+  <Suppression>
+    <DiagnosticId>CP0003</DiagnosticId>
+    <Target>System.Reflection.DispatchProxy, Version=4.0.6.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51</Target>
+    <Left>lib/netstandard2.0/System.Reflection.DispatchProxy.dll</Left>
+    <Right>lib/netstandard2.0/System.Reflection.DispatchProxy.dll</Right>
+    <IsBaselineSuppression>true</IsBaselineSuppression>
+  </Suppression>
+  <Suppression>
+    <DiagnosticId>CP0003</DiagnosticId>
+    <Target>System.Reflection.DispatchProxy, Version=4.0.6.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51</Target>
+    <Left>ref/net461/System.Reflection.DispatchProxy.dll</Left>
+    <Right>lib/netstandard2.0/System.Reflection.DispatchProxy.dll</Right>
+    <IsBaselineSuppression>true</IsBaselineSuppression>
+  </Suppression>
+  <Suppression>
+    <DiagnosticId>CP0003</DiagnosticId>
+    <Target>System.Reflection.DispatchProxy, Version=4.0.6.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51</Target>
+    <Left>ref/netstandard2.0/System.Reflection.DispatchProxy.dll</Left>
+    <Right>lib/netstandard2.0/System.Reflection.DispatchProxy.dll</Right>
+    <IsBaselineSuppression>true</IsBaselineSuppression>
+  </Suppression>
+  <Suppression>
+    <DiagnosticId>CP0003</DiagnosticId>
+    <Target>System.Reflection.DispatchProxy, Version=4.0.6.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51</Target>
+    <Left>ref/uap10.0.16299/System.Reflection.DispatchProxy.dll</Left>
+    <Right>lib/netstandard2.0/System.Reflection.DispatchProxy.dll</Right>
+    <IsBaselineSuppression>true</IsBaselineSuppression>
+  </Suppression>
+  <Suppression>
+    <DiagnosticId>PKV006</DiagnosticId>
+    <Target>.NETCore,Version=v5.0</Target>
+  </Suppression>
+  <Suppression>
+    <DiagnosticId>PKV006</DiagnosticId>
+    <Target>.NETStandard,Version=v1.3</Target>
+  </Suppression>
```

Comment:
Add these TFMs to this list instead: https://github.com/dotnet/maintenance-packages/blob/92eba09e9ec1004fe6d3db4f4e73f258b5d02579/eng/packaging.targets#L11

This makes package validation not validate api compatibility for these out-of-support TFMs.

Reply:
`net5.0` was already in the packaging.targets file in main:
https://github.com/dotnet/maintenance-packages/blob/92eba09e9ec1004fe6d3db4f4e73f258b5d02579/eng/packaging.targets#L16

If I remove the suppresion and enable IsPackable=true in the src csproj, why do I still get the API Compat error then?:

```
C:\Users\calope\.nuget\packages\microsoft.dotnet.apicompat.task\9.0.100-alpha.1.23560.11\build\Microsoft.NET.ApiCompat.ValidatePackage.targets(39,5): er
ror PKV006: Target framework .NETCore,Version=v5.0 is no longer supported in the latest version. [D:\maintenance-packages\src\System.Reflection.Dispatch
Proxy\src\System.Reflection.DispatchProxy.csproj]
```

Reply:
`.NETCore,Version=v5.0` is not `net5.0`, it's `netcore50` which was used for UWP applications.

Reply:
Thanks for that. I submitted it in a separate PR: https://github.com/dotnet/maintenance-packages/pull/51